poveste.txt

```diff
@@ -0,0 +1,13 @@
+Eram adunați într-un mănunchi de cântec mut. Fără sunet, vocile noastre se sublimau sub tensiunea care ne strângea tâmplele, străpungeau cu greu membranele individuale și se adunau undeva deasupra într-un nor de tensiune. 
+
+Strofă, refren, strofă. Cântec, reculegere, cântec. Un ritm familiar.
+
+Din tristeți izolate, să împletim duioșii care să ne lege.
+Din amintiri, văi indundate, să facem lunci fertile.
+
+Nu era ușor, dar ritmul era familiar. Nu eram prezent doar la îndeplinirea de atunci a ritualului. Eram prezent la cumulul tuturor îndeplinirilor ritualului de până atunci, și asta aducea puțină liniște. Știam că un cerc perfect se închide. Și mai mult, știam că ciclul se reia. Nu prin reîncarnare, ci prin oameni care cred în ce a crezut și ea. Prin oameni care iubesc ce a iubit ea.
+Ritualul e mereu același: după cântec, scoatem la licitație moștenirea. Moștenirea nu e un beneficiu. E o răspundere. Moștenirea nu e ce avea, ci e ce credea, ce făcea și ce iubea ea. Fiecare dintre noi trebuie să adune o listă și să vândă fiecare punct din listă cât mai bine. Să facă munca și iubirea ei nu numai cunoscute, ci și înțelese, și însușite, și râvnite. Fiecare ființă, grup, sau lucru pe care ea l-a iubit, cel puțin unul dintre noi trebuie să și-l ia ca răspundere, să promită să-l  continue sau să-l păzească, să-l iubească și să-l împlinească.
```

Comment:
> Moștenirea nu e ce avea, ci e ce credea, ce făcea și ce iubea ea. 

In, for the lack of a better word, filozofia mea, « a avea » are o conotatie extrem de pozitiva. Nu poti iubi cu adevarat ceva ce nu e al tau. Sau, daca vrei, a avea ajunge sa se identifice cu restul (a crede, a face, a iubi). Sau restul ajunge sa se identifice cu a avea. Cu toate ca entitatile despre care vorbesc isi pastreaza individualitatea. Nu sunt confortabil in a scrie ceva ce « saraceste » verbul « a avea ».  ; de exemplu,  « Moștenirea e ce credea, ce făcea și ce iubea ea. »  ar fi o reformulare prin omitere, cu care sunt ok, care pastrea oarecum si ideea ta. Poti propune alte variante.

In, for the lack of a better word, filozofia mea, « a avea » are o conotatie extrem de pozitiva. Nu poti iubi cu adevarat ceva ce nu e al tau. Sau, daca vrei, a avea ajunge sa se identifice cu restul (a crede, a face, a iubi). Sau restul ajunge sa se identifice cu a avea. Cu toate ca entitatile despre care vorbesc isi pastreaza individualitatea. Nu sunt confortabil in a scrie ceva ce